data/versions.yaml

```diff
@@ -4,8 +4,8 @@
 #
 
 # Note: When updating cucumber-jvm version, please update both the snapshot version and the regular version
-cucumberjvm-snapshot: "4.4.0-SNAPSHOT"
-cucumberjvm: "4.3.1"
+cucumberjvm-snapshot: "4.5.5-SNAPSHOT"
```

Comment:
I'd like to at least merge the version updates (if not the import statements just yet). Would you mind separating these changes into 2 different PRs? (We could update import statements when v5 hits? And we'd need to make sure to update consistently all throughout the docs)

Reply:
Better make a new PR then. We're at 4.7.1 right now.

Reply:
With 5.0.0-SNAPSHOT as the next planned release.

Reply:
Good to know, thnx :)